app/controllers/courses_controller.rb

```diff
@@ -185,6 +188,11 @@ def set_course
     protect_privacy
   end
 
+  def set_courses_wikis
```

Comment:
I don't think this is necessary; you can get the wikis from the `course`, just with `@course.wikis` (and Rails will cache that on the Course object so that if you need to call it more than once, it won't result in any extra queries).